AtomAnu/MEng-V2I
# Simple charging recommendation

Description of Work:

- Development of the first charging recommendation algorithm. This algorithm is created in a class called `charging_recommendation` in  `charging_recommendation.py`

Fixes #11 

Testing Instructions
Run the example code in 'main.py'. Try to adjust the inputs to see different results
